bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -553,4 +573,17 @@
 		<description>Name of the scenario to trigger</description>
 	</channel-type>
 
+	<channel-type id="bypass-state">
+		<item-type>Switch</item-type>
+		<label>Bypass State</label>
+		<description>Indicates whether the device is currently bypassed.</description>
+		<state readOnly="true">
+			<options>
```

Comment:
Question @jlaur: I saw that some channel-types with `<item-type>Switch</item-type>` in our `thing-types.xml` have `<options>`, so I also added options here. But does this actually make sense for a switch?

This channel recieves the states `ON`, `OFF` or `UNDEF`, which are mapped to the Bosch-specific states `BYPASS_ACTIVE`, `BYPASS_INACTIVE` and `UNKNOWN`.

Reply:
> But does this actually make sense for a switch?

Yes, it does make sense. For a read-only channel as this, you will be able to see the option texts. See for example:

https://github.com/openhab/openhab-addons/blob/af13c9d133753454ef222520ed2517125e835c3b/bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml#L149-L160

In the UI:

![image](https://github.com/openhab/openhab-addons/assets/19519842/7d9cf6bd-497c-48dc-a891-3537ddeef357)

I actually also found examples within the boschshc binding, but I think they are invalid, because they don't map ON/OFF for the Switch item:

https://github.com/openhab/openhab-addons/blob/af13c9d133753454ef222520ed2517125e835c3b/bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml#L358-L380

Maybe I'm wrong and ENABLED/DISABLED is also mapped. Perhaps you could verify?

Reply:
I suspected that only `ON`/`OFF` makes sense here. I corrected the options in this PR. Can I also add `UNDEF` as option here?

I will fix the other switch options in a separate PR.

Reply:
I just verified that the UI does not display the option text for `UNDEF`, only for `ON` and `OFF`. Now we have two options:

* we consider this a bug and leave the option `UNDEF` in the XML
* we remove the option `UNDEF`

What would you propose in this case?

Reply:
I can see that no other binding defines an undef option, so I would propose to remove it, since it doesn't have the desired effect anyway.